### PR TITLE
Feature/jquery selectors

### LINKED ISSUE #24: Remove one-time use selectors from file, replace with jQuery selectors

### LINKED ISSUE #25: Turn any re-used global selectors into jQuery

### PR DESCRIPTION
**Pull request by edignot**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling- no new features

### Detailed Description
- Changed all querySelectors to jQuery selectora. Deleted all global variables
### Why is this change required? What problem does it solve?
- 
### Were there any challenges that arose while implementing this feature? If so, how were the addressed?
- #25
- #24
### Files modified:
- [x] index.html
- [x] scripts.js.css

### CODE REVIEW

**Comment by edignot** [comment]
I included jQuery from a CND, should we still download local jQuery library for this project?

**Comment by rubyr** [comment]
Yes, I'd probably keep a local copy on hand just in case, using the method they showed in the jquery lesson.

File: src/scripts.js [hunk 1]
```diff
@@ -85,21 +53,22 @@ function getUserById(id, listRepo) {
 
 
 function addInfoToSidebar(user, userStorage) {
-  sidebarName.innerText = user.name;
-  headerText.innerText = `${user.getFirstName()}'s Activity Tracker`;
-  stepGoalCard.innerText = `Your daily step goal is ${user.dailyStepGoal}.`
+  $('#sidebarName').text(user.name);
+  $('#headerText').text(`${user.getFirstName()}'s Activity Tracker`);
+  $('#stepGoalCard').text(`Your daily step goal is ${user.dailyStepGoal}.`);
+  // avStepGoalCard is never selected and used
```
Comment (rubyr) [inline src/scripts.js]: If it's never selected, then go ahead and remove the next line (and this comment, too)

File: src/index.html [hunk 2]
```diff
@@ -1,9 +1,11 @@
 <!DOCTYPE html>
 <html>
+
```
Comment (rubyr) [inline src/index.html]: remove this whitespace

File: src/index.html [hunk 3]
```diff
@@ -191,6 +193,8 @@ <h2 class="sidebar-header-name" id="sidebarName">FRIENDLY STEP CHALLENGE!</h2>
       </section>
     </section>
   </section>
+  <script src="https://code.jquery.com/jquery-3.4.1.min.js"
+    integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
 </body>
```
Comment (rubyr) [inline src/index.html]: remove this whitespace, too

**Review by rubyr — COMMENTED** [review]
Overall, really good work on this! Just a few things might need to be adjusted before merging to master.

**Review by rubyr — APPROVED** [review]

**Pull request merged**